.eslintrc

```diff
@@ -64,11 +71,18 @@
         "annotateUndefined": "never"
       }
     ],
-    "flowtype/require-valid-file-annotation": 0,
+    "flowtype/require-valid-file-annotation": [
+      1,
```

Comment:
I wonder if we could ignore `__test__` directories for this rule. I don't particularly care about specs having flowtypes.

Reply:
I can add an `.eslintrc` to all the test dirs that overrides just this

Reply:
That would work, but I think a glob override in the project config is easier to do

https://eslint.org/docs/user-guide/configuring#configuration-based-on-glob-patterns

Reply:
done